composer.json

```diff
@@ -26,7 +26,8 @@
         "source": "https://github.com/FriendsOfFlarum/nightmode"
     },
     "require": {
-        "flarum/core": "^0.1.0-beta.8"
+        "flarum/core": "^0.1.0-beta.8",
+        "fof/components": ">=0.1.1"
```

Comment:
We need to call `AddFofComponents` extender from `extend.php` to make sure they are added.

https://github.com/FriendsOfFlarum/prevent-necrobumping/blob/cfe4580e570bf38b5cbaf9584e61dfd96f2212b9/extend.php#L28-L30

Reply:
Thanks for letting me know -- would have had no clue otherwise :P